docs/snippets/vue/addon-highlight-reset.js.mdx

```diff
@@ -0,0 +1,28 @@
+```js
+// MyComponent.stories.js
+
```

Comment:
Do these Vue snippets apply to both Vue 2 & Vue 3?

Reply:
Yes, they do, when testing this out, I've used both Vue 2 and Vue 3 and TS variations to vet if there weren't any issues that required additional documentation.